src/types/types.ts

```diff
@@ -2591,6 +2591,93 @@ export interface DebugPrestate {
   storage: Record<string, string>;
 }
 
+/*
+ * SDK Representation of a user operation and bundler types
```

Comment:
since this type is going to live with a bunch of other types, can you explain this in the context of the bundler namespace. ex:
```
Representation of a submitted user operation returned by the {@link BundlerNamespace.getUserOperationByHash} 
```